os/binfmt/binfmt.h

```diff
@@ -148,6 +151,51 @@ void binfmt_freeargv(FAR struct binary_s *bin);
 #define binfmt_freeargv(bin)
 #endif
 
+
+#ifdef CONFIG_ARM_MPU
+static inline void binfmt_mpuinit(struct binary_s *binp)
+{
+	uint32_t *regs = NULL;
+	uint8_t nregion = 0;
+
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
+	if (binp->islibrary) {
+		regs = binp->cmn_mpu_regs;
+		nregion = mpu_get_nregion_info(MPU_REGION_COMMON_BIN);
+	} else
```

Comment:
`else` looks strange.
I think you intended two cases as below:
1. remove `#else`
```
#ifdef config_common_bin
   if (islibrary) {
       do common bin
   } else
#endif 
   {
       do app bin
   }
```
2. remove `else`
```
#ifdef config_common_bin
   if (islibrary) {
       do common bin
   } 
#else 
   {
       do app bin
   }
#endif
```

I think 1st is correct. Could you check this?